docker-compose.yaml

```diff
@@ -3,14 +3,13 @@ version: '2'
 services:
   app:
     build: ./
+    image: opencart-2.3.x
     restart: always
     ports:
       - 8080:80
-    environment:
-      OC_ROOT_DIR: /var/www/html
     volumes:
       - app:/var/www/html
```

Comment:
I would prefix for volumes as in [2.2.x version](https://github.com/sendsmaily/smaily-opencart-module/pull/38#pullrequestreview-431362012)

Reply:
Added a suffix as explained in 2.2.x branch.